e2e/scripts/st_multiselect.py

```diff
@@ -16,19 +16,23 @@
 
 import streamlit as st
 
+session_state_enabled = st.checkbox("session_state_enabled")
+if "multiselect 9" in st.session_state and session_state_enabled:
+    st.session_state["multiselect 9"] = ["male", "female"]
```

Comment:
nit: would be nicer to have this checkbox right above the multiselect it affects assuming it doesn't mess with the test somehow

Also, I think `"session_state_enabled"` isn't particularly descriptive of what this is for (the first thing I thought of when I read it was that you were turning the session state feature on/off) and could be renamed something like "set_multiselect_9"